src/serac/coefficients/tests/CMakeLists.txt

```diff
@@ -0,0 +1,28 @@
+# Copyright (c) 2019-2021, Lawrence Livermore National Security, LLC and
+# other Serac Project Developers. See the top-level LICENSE file for
+# details.
+#
+# SPDX-License-Identifier: (BSD-3-Clause) 
+
+if (ENABLE_GTEST)
+
+    set(test_dependencies serac_physics serac_coefficients test_utils)
```

Comment:
Are these true at this point? since i assume they were just copy/pasted from the original place...

Reply:
I know physics and coefficients are needed. I'll check test utils.

Reply:
Physics and test_utils were both needed. Coefficients has been removed.